platform/mbed_wait_api.h

```diff
@@ -55,9 +55,9 @@ extern "C" {
  *  @param s number of seconds to wait
  *
  *  @note
- *    If the RTOS is present, this function always spins to get the exact number of microseconds,
- *    which potentially affects power (such as preventing deep sleep) and multithread performance.
- *    You can avoid it by using Thread::wait().
+ *    If the RTOS is present, this function spins to get the exact number of microseconds,
```

Comment:
Microsecond precision is no longer maintained when a delay of greater than 10ms is used. In this case the tolerance would be `wait_time - 500us` to `wait_time +500us`